test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -2214,21 +2222,18 @@ describe('the rubicon adapter', function () {
           expect(imp.video.w).to.equal(640);
           expect(imp.video.h).to.equal(480);
           expect(imp.video.pos).to.equal(1);
-          expect(imp.video.context).to.equal('instream');
```

Comment:
I noticed this is not passed any longer:

`request.imp.video.context`

Let's ask and make sure this is ok to leave out.

Reply:
In issue HB-15147 see @bretg comment from 10/01/2022 

> imp.video.context - not valid ORTB

Reply:
👍